assets/js/common.js

```diff
@@ -28,7 +29,7 @@ var $gp = (
 					$gp.notices.element.stop( true, true );
 
 					if ( true === dismissable ) {
-						dismissButton = ' <button type="button" class="button is-link gp-js-message-dismiss">' + $gp.l10n.dismiss + '</button>';
+						dismissButton = ' <button type="button" class="button is-link gp-js-message-dismiss">' + __( 'Dismiss', 'glotpress' ) + '</button>';
```

Comment:
```suggestion
						dismissButton = ' <button type="button" class="button is-link gp-js-message-dismiss">' + wp.i18n.__( 'Dismiss', 'glotpress' ) + '</button>';
```

No need for an extra variable.